### ISSUE TITLE
异常: System.ArgumentOutOfRangeException

### ISSUE THREAD

**Issue by lzw-723** [PowerShell/PSReadLine#1467]
```
PS E:\JavaProject\MusicFx>  & 'C:\Users\lzw-7\.vscode\extensions\vscjava.vscode-java-debug-0.25.1\scripts\launcher.bat' 'F:\Program Files\Java\jdk1.8.0_241\bin\java.exe' '-agentlib:jdwp=transport=dt_socket,server=n,suspend=y,address=localhost:8602' '-Dfile.encoding=UTF-8' '-cp' 'C:\Users\lzw-7\AppData\Local\Temp\cp_dv4mjd7r9s02dg62fyb65l6zr.jar' 'io.github.lzw.MainApp'
哎呀，出现问题。请在报告此 Bug 时添加以下详细信息。
在 GitHub 上报告: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
上 200 个密钥:
 g e n t l i b : j d w p = t r a n s p o r t = d t _ s o c k e t , s e r v e r = n , s u s p e n d = y , a d d r e s s = l o c a l h o s t : 8 6 0 2 ' Space ' - D f i l e . e n c o d i n g = U T F - 8 ' Space ' - c p ' Space ' C : \ U s e r s \ l z w - 7 \ A p p D a t a \ L o c a l \ T e m p \ c p _ d v 4 m j d 7 r 9 s 0 2 d g 6 2 f y b 6 5 l 6 z r . j a r ' Space ' i o . g i t h u b . l z w . M a i n A p p ' Space Enter


异常:
System.ArgumentOutOfRangeException: 该值必须大于或等于零，且必须小于控制台缓冲区在该维度的大小。  
参数名: top
实际值是 -2。
   在 System.Console.SetCursorPosition(Int32 left, Int32 top)
   在 Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   在 Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   在 Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   在 Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   在 Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   在 Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   在 Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

**Comment by iSazonov** [comment]
See #1306

**Comment by daxian-dbw** [comment]
This issue was fixed (see #1306). Please upgrade to the 2.0.1 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions.
